scripts/route_check.py

```diff
@@ -120,22 +135,64 @@ def get_routes():
         if not is_local(k):
             valid_rt.append(add_prefix_ifnot(k.lower()))
 
-    print_message(syslog.LOG_DEBUG, json.dumps({"ROUTE_TABLE": sorted(valid_rt)}, indent=4))
+    print_message(syslog.LOG_DEBUG, json.dumps(
+        {"ROUTE_TABLE": sorted(valid_rt)}, indent=4))
     return sorted(valid_rt)
 
 
+def strip_ip_asic_route_entry(k):
+    return k.lower().split("\"", -1)[3]
+
+
+def upd_handler(key, data):
+    db = thread_data[THREAD_DB]
+    if not db:
+        print_message(syslog.LOG_DEBUG, "subscriber thread for ASIC_DB:{} exiting".
+                format(ASIC_TABLE_NAME))
+        sys.exit(0)
+    key = ConfigDBConnector.deserialize_key(key)
+    if key.startswith(ASIC_TABLE_KEYVAL):
+        keys = db.get_keys(ASIC_TABLE_NAME)
+        if key in keys:
+            thread_data[THREAD_SET].append(strip_ip_asic_route_entry(key))
+        else:
+            thread_data[THREAD_DEL].append(strip_ip_asic_route_entry(key))
+        print_message(syslog.LOG_DEBUG, "key={} op={}".format(
+            key, "set" if key in keys else "del"))
+
+
+def subs_thread():
+    db = thread_data[THREAD_DB]
+    db.subscribe(ASIC_TABLE_SUBS, lambda table,
+            key, data: upd_handler(key, data))
```

Comment:
@renukamanavalan we should swsscommon here so that we can handle the operation in batch. This has been done in caclmgrd where we select and pop all the operation until empty. swsscommon will also provvide op as SET or DEL

Reply:
The systems running 20191130.56 does not have swsscommon. Do the later versions support ?

```
remanava@BN6-0101-0123-15T1:~$ python3
Python 3.5.3 (default, Nov 18 2020, 21:09:16) 
[GCC 6.3.0 20170516] on linux
Type "help", "copyright", "credits" or "license" for more information.
>>> from swsscommon import swsscommon
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
ImportError: No module named 'swsscommon'
>>> quit()
remanava@BN6-0101-0123-15T1:~$ show version| grep -e "^SO"
SONiC Software Version: SONiC.20191130.56
remanava@BN6-0101-0123-15T1:~$ 

```

Reply:
swsscommon is available in python2
Let me switch to it. We would not need a thread.